src/librustdoc/clean/inline.rs

```diff
@@ -575,6 +548,10 @@ fn separate_supertrait_bounds(mut g: clean::Generics)
 }
 
 pub fn record_extern_trait(cx: &DocContext, did: DefId) {
+    if did.is_local() {
+        return;
```

Comment:
You're not constant about your `return` calls: some have `;`, some others don't. ;)

Reply:
The `return` lines without semicolons are ones i didn't write myself. :P

Reply:
The diff doesn't agree. :p